taskfiles/k8s.yml

```diff
@@ -310,3 +318,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - helm dep update charts/operator
+      # To change name of the operator helm chart malformer program is executed
+      - go run ./nightly-malformer/...
+      - helm package charts/operator
```

Comment:
nit: To avoid littering the repo when running this task, it'd be preferable to output the build to either a tempdir or the `dist` directory which is git ignored.

Reply:
Agree. will adjust.

My personal goal would be to use go program to load chart and the push it to docker hub. I'm using helm cli just for the quick turnaround.

Reply:
Oh noo. Because of docker hub multi-platform problem I can not avoid littering :( 
```
ERROR: Multi-platform build is not supported for the docker driver.
--
  | Switch to a different driver, or turn on the containerd image store, and try again.
  | Learn more at https://docs.docker.com/go/build-multi-platform/
```
https://buildkite.com/redpanda/redpanda-operator/builds/3585#0193a9ae-d78e-4e80-9e72-b32d16f66507/325-328